static/js/containers/ChannelPage.js

```diff
@@ -101,20 +61,37 @@ class ChannelPage extends React.Component {
       )
     }
   }
+}
 
-  render() {
-    const { channels, postsForChannel } = this.props
-    return <Loading restStates={[channels, postsForChannel]} renderContents={this.renderContents} />
-  }
+const shouldUpdateRequirements = R.complement(R.eqProps("channelName"))
```

Comment:
Ah, right. If we prefer the brevity, I'm OK with keeping it, but then I think a comment is even more important